zebra/zebra_evpn_mac.c

```diff
@@ -254,6 +255,37 @@ static void zebra_evpn_mac_get_access_info(zebra_mac_t *mac,
 	}
 }
 
+#define MAC_BUF_SIZE 256
+static char *zebra_evpn_zebra_mac_flag_dump(struct zebra_mac_t_ *mac, char *buf,
+					    size_t len)
+{
+	if (mac->flags == 0) {
+		snprintfrr(buf, len, "None ");
+		return buf;
+	}
+
+	snprintfrr(
+		buf, len, "%s%s%s%s%s%s%s%s%s%s%s%s",
+		CHECK_FLAG(mac->flags, ZEBRA_MAC_LOCAL) ? "LOC " : "",
```

Comment:
Could we have a comment in zebra_evpn_mac.h explaining that the outputter exists and needs to be maintained as flags are added?

Reply:
This is a good point - these flags are also displayed in bgp_evpn.c and so having the utility in lib would enable it's use universally.

Reply:
fixed